site/content/en/docs/tasks/setup_multikueue.md

```diff
@@ -0,0 +1,197 @@
+---
+title: "Setup a MultiKueue environment"
+date: 2024-02-26
+weight: 9
+description: >
+  Additional steps needed to setup the multikueue clusters.
+---
+
+Check the concepts section for a [MultiKueue overview](/docs/concepts/multikueue/). 
+
+## Worker Cluster
+
+When MultiKueue dispatches a workload from the manager cluster to a worker cluster, it expects that the job's namespace and LocalQueue also exist in the worker cluster.
+In other words, you should ensure that the worker cluster configuration mirrors the one of the manager cluster in terms of namespaces and LocalQueues.
+
+To create the sample queue setup in the `default` namespace apply:
+
+{{< include "examples/admin/single-clusterqueue-setup.yaml" "yaml" >}}
+
+### MultiKueue Specific Kubeconfig
+
+In order to delegate the jobs in a worker cluster, the manager cluster needs to be able to create, delete, and watch workloads and their parent Jobs.
+
+Follow these steps to create a Kubeconfig restricted to this use case.
+
+1. Create a service account, a restricted role and role binding.
+
+```bash
+MULTIKUEUE_SA=multikueue-sa
```

Comment:
Is this the only reason we have it as a batch script instead of plain yaml file?

Reply:
Do we realistically expect users to often have a different value here?